presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/api/ConnectorsAPI.java

```diff
@@ -33,38 +34,64 @@
 import java.nio.file.Paths;
 
 @Path("/connectors")
-@Api(description = "the connectors API")
-@javax.annotation.Generated(
-        value = "io.swagger.codegen.languages.JavaJAXRSSpecServerCodegen",
-        date = "2017-06-23T09:53:13.549-04:00")
+@Api(description = "API to read and write to Presto connector files")
 @Singleton
 public final class ConnectorsAPI
 {
     private static final APIFileHandler apiFileHandler = new APIFileHandler(Paths.get("/etc/presto/catalog"));
 
     @GET
-    @Produces({MediaType.TEXT_PLAIN})
+    @Produces(MediaType.TEXT_PLAIN)
     @ApiOperation(value = "Get avaiable connector file names")
     @ApiResponses(value = {
-            @ApiResponse(code = 200, message = "Retrieved configuration", response = String.class)})
+            @ApiResponse(code = 200, message = "Retrieved configuration")})
     public synchronized Response getConnectors()
     {
         return apiFileHandler.getFileNameList();
     }
 
     @GET
     @Path("/{file}")
-    @Produces({MediaType.TEXT_PLAIN})
-    @ApiOperation(value = "Get connectors by file")
+    @Produces(MediaType.TEXT_PLAIN)
+    @ApiOperation(value = "Get contents of connector file")
```

Comment:
"Get a catalog configuration file" -- this would be somewhat clearer if it were /{catalog}, but that is a bit of an overarching change